packages/react-hooks/src/hooks/debounced.ts

```diff
@@ -0,0 +1,29 @@
+import {useEffect, useRef, useState} from 'react';
+
+export const DELAY_COMMIT_SETTING_CHANGE = 500;
+
+export function useDebouncedValue<T>(
+  value: T,
+  delay = DELAY_COMMIT_SETTING_CHANGE,
```

Comment:
Would really be great to use the same signature as `useDeferredValue` so it's easier to migrate in the future... https://reactjs.org/docs/concurrent-mode-reference.html#usedeferredvalue